cypress/e2e/tests/setup/rancher-setup.spec.ts

```diff
@@ -45,4 +45,9 @@ describe('Rancher setup', () => {
     //   cy.url().should('equal', `${ Cypress.config().baseUrl }/home`);
     // });
   });
+
+  it('Create standard user after login', () => {
```

Comment:
@richard-cox You are correct, part of the reason was to avoid plumbing in the user's credentials but the main reason is that we create a standard user via api request which relies on the `login()` command. In the `login()` command, a session is created, and that session token is then stored and used to make api request which creates a standard user. It makes sense to put `createUser()` here because at this point Rancher is set up and the UI is accessible to the admin so the admin is able to login and grab the session token. 
I did try to put `createUser()` in `cypress/support/e2e.ts` in a `before()` hook (so that it runs before all test) but it failed because it would precede Rancher setup spec. Ideally, we wouldn't have to rely on the `login()` command to create the standard user but I haven't found another way to get a session token to send requests.
Also, an idea to handle to parallelization concern: we could pull Rancher setup spec out of the admin test run and put it in its own step, that way the setup step would precede both the admin and standard user test steps and they can run in parallel.

Reply:
i like that idea of having a 'setup' style initial step then running other tests in parallel. Can you create an issue in `rancher/dashboard` to track?

Reply:
yessir, issue created for tracking: https://github.com/rancher/dashboard/issues/9243